src/main/webapp/index.html

```diff
@@ -67,19 +62,33 @@
         </div>
         <div id="comment-slider">
           <p>Maximum number of comments: </p>
-            <input type="range" id="num-comments-input" name="num-comments-input" min="100" max="1000" value = "100" step="10"/>
-            <small id="slider-output" name = "slider-output">100</small>
-        </div>   
+            <input type="range" id="num-comments-input" name="num-comments-input" min="100" max="500" value = "100" step="10"/>
```

Comment:
In another PR do you think we could reject requests on the backend that request more than 500 comments?

Reply:
Do you mean like having the back end deal with having too many comments? Definitely possible but I'm confused on why it'd be necessary if the user is limited on the front end in the first place

Reply:
So even though the UI limits the user from selecting more than 500 comments in the slider, nothing prevents them from manually updating the HTML in their browser to change the slider value to anything that they want.

Generally it's good practice to place most request validation on the backend, since most front-end validation can be trivially bypassed.

Here's an article about this:
https://medium.com/better-programming/back-end-data-validations-73ea9004c6d7

Reply:
Oh makes sense! Will make this change in another pr